miqi/runtime/turn_runner.py

```
@@ -405,6 +405,128 @@ async def _drain_steer_messages() -> list[dict[str, Any]]:
+                        seen_names.append(tc.name)
+                turn._plan_phases = phases
+                turn._plan_seen_tools = set(seen_names)
+                from miqi.execution.task_policy import (
+                    should_plan_confirm,
+                    should_show_timeline,
+                    tool_risk,
+                )
+                policy = getattr(turn, "execution_policy", "edit")
+                if "ask_user_plan_confirm" not in seen_names:
+                    if policy == "auto":
+                        # GPT P0-3：Auto 模式非阻塞 Timeline（复杂任务展示——
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)